src/scipp/core/operations.py

```diff
@@ -356,3 +356,18 @@ def _merge_data_group(lhs: DataGroup, rhs: DataGroup) -> DataGroup:
             raise DatasetError(f"Cannot merge data groups. Mismatch in item {k}")
         res[k] = v
     return res
+
+
+def label_based_to_positional_index(
```

Comment:
Was it in purpose that you used a different name than in the binding?

Reply:
I think the double `index` was a bit unnecessary. I'll update the name in the c++ code.